### DISCUSSION TITLE
Алкоголь.

### DISCUSSION THREAD

**Discussion by Asdasdasdasdsaddczx2** [octothorp-team/public-dobrograd#765] [Предложения]
Привет! И так начнём с моего предложения, а именно увиличить механ длительность эффекта алкоголя ХОТЯБЫ на 10 минут дольше, вы спросите почему? Ну во первых я уже пробовал алкоголь в ИРЛ, после 1 или 2 бутылки, начинает кружиться голова и нормально говорить будет очень тяжело, ты будешь запинаться и терять речь, а с нынищнем эффектом ты просто 2-4 минуты походишь и все пройдёт, я как офицер DPD могу сказать что ситы с тем как вы пьяны за рулём, очень классные, поэтому алкоголь надо хоть как то улучшить.

**Comment by travka4000** [comment]
Я за, ведь действительно, знаешь как будто персонаж трезвяк

**Reply by travka4000** [reply]
что эму плевать скок он выпил
